include/swift/AST/Types.h

```diff
@@ -2654,6 +2658,13 @@ class AnyFunctionType : public TypeBase {
 
     /// Whether the parameter is marked 'owned'
     bool isOwned() const { return Flags.isOwned(); }
+
+    bool operator!=(Param const &b) const {
```

Comment:
Can you add `==` here as well, and then implement `!=` in terms of that?

Reply:
Sorry I didn't know that it was a convention, will fix right away!